{
  Utils.<RuntimeException>doThrow0(t);
}
